src/BalanceLaws/boundaryconditions.jl

```diff
@@ -0,0 +1,3 @@
+function boundary_conditions end
```

Comment:
Can we add some docs to this? It would be helpful to also know what this does in edge cases like `boundary_conditions(::BalanceLaw) = ()` and `boundary_conditions(::BalanceLaw) = (nothing,)`

Reply:
Maybe add an entry in `docs/src/APIs/BalanceLaws/BalanceLaws.md`, too.